src/controllers/rest/impl/security/AdminController.ts

```diff
@@ -59,6 +60,23 @@ export class AdminController extends AbstractAdminController implements IAdminCo
         };
     }
 
+    @Get("/getBucketType/:token")
+    public async getBucketType(@PathParams("token") token: string): Promise<BucketType | string> {
+        return (await this.userAdminService.getBucketType(token)) ?? "NOTFOUND";
+    }
+
+    @Post("/setBucketType")
+    public async setBucketType(
+        @Res() res: PlatformResponse,
+        @Required() @BodyParams("token") token: string,
+        @Required() @BodyParams("bucketType") bucketType: BucketType,
+    ): Promise<PlatformResponse> {
+        if (await this.userAdminService.setBucketType(token, bucketType)) {
+            return super.doSuccess(res, "Bucket Type Set Succeeded");
+        }
+        return super.doError(res, "Bucket Type Set Failed", 500);
```

Comment:
we don't use `500` we have constants for https statuses `StatusCodes.INTERNAL_SERVER_ERROR`